lib/src/rules/prefer_if_elements_to_map_literals.dart

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r'Prefer if elements to map literals where possible.';
```

Comment:
"map literals"? The lint flags uses of the constructor `Map.fromIterable`.

Reply:
Absolutely; sloppy copy-paste. 😬